flaskbb/forum/views.py

```diff
@@ -1060,182 +1061,215 @@ def post(self, post_id):
         return redirect(post.topic.url)
 
 
+class MarkdownPreview(MethodView):
+    decorators = [login_required]
```

Comment:
Do we want to prevent guests doing previews? We could use a CSRF token to prevent being a markdown rendering API.